Examples/UIExplorer/PanResponderExample.js

```diff
@@ -78,13 +80,17 @@ var PanResponderExample = React.createClass({
 
   _highlight: function() {
     this.circle && this.circle.setNativeProps({
-      backgroundColor: processColor(CIRCLE_HIGHLIGHT_COLOR)
+      style: {
+        backgroundColor: processColor(CIRCLE_HIGHLIGHT_COLOR)
+      }
     });
   },
 
   _unHighlight: function() {
     this.circle && this.circle.setNativeProps({
-      backgroundColor: processColor(CIRCLE_COLOR)
+      style:{
```

Comment:
For reference, after running `npm run lint`

![screen shot 2015-10-09 at 7 05 54 pm](https://cloud.githubusercontent.com/assets/656630/10407452/c90eedcc-6eb8-11e5-8940-68cefb0525d6.png)